### DISCUSSION TITLE
Locales are ignored?

### DISCUSSION THREAD

**Discussion by Stephan-4711** [lavinir/hassio-onedrive-backup#175] [Ideas]
Hi,
Thanks for the great addon.
I'm just in a hassel with the promted dates in your addon.
It seems the german locales for the time and date formates are ignored and shown in US format.
Is there a way to change this?

**Comment by paulsgale** [comment]
I second this - no US format dates please!

**Comment by lavinir** [comment]
Started looking into this. Weirdly I'm not seeing the correct API to fetch the User's locale from Home Assistant.

**Comment by novakreo** [comment]
Would it be possible to add a manual setting for now?

**Comment by Adrian57** [comment]
Yes please. System Locales. In the heat of a panic restore, it is easy to mistake the date 2/1/2024 in the UK for the 2nd of January. It is in the US but not here!

**Comment by lavinir** [comment]
I think I'll add an option to override the date time format so the user can just set whatever they want.

**Comment by SteveXNZ** [comment]
Adding a date format option in Settings would be good if there's no way to query the locale.  mm/dd vs dd/mm ambiguity can easily lead to errors.  Personally I prefer dd/mmm to remove all doubt.

**Comment by lavinir** [comment]
This will be tracked with #272
